README.md

```diff
@@ -0,0 +1,195 @@
+## Directory Structure
```

Comment:
Suggest to add "Introduction" section. This section should contain brief description of repository (what this repository is about, the aim and purpose of this repository, etc...). Thus, user will have general overview of repo before moving to the details described in next sections.